update flagType selector for new timeline changes

Vogel612 opened the pull request:
The updated timeline does no longer use ".event-verb", but uses the nonsensical class ".wmn1".
Instead of selecting by type, we're selecting the third td, which could be more resilient against future changes.

Vogel612 committed:
update flagType selector for new timeline changes

The updated timeline does no longer use ".event-verb", but uses the nonsensical class ".wmn1".
Instead of selecting by type, we're selecting the third td, which could be more resilient against future changes.

Shog9 reviewed (approved):
Should be fine; @catija would you mind testing when you get a chance? Pretty sure you still have commit rights here, if not I'll add.

Vogel612 commented:
For what it's worth, the timeline update seems to have also broken determining the flag status as handled. I should've checked a bit more carefully for uses of the removed class, apparently...

Vogel612 committed:
Fix handled flags and reviews

Deletion events are no longer handled with `.deletion-event-details` but just `.deletion-events`. 

This commit also cleans up some leftovers from the preceding timeline-fix

Vogel612 commented:
Even with the additional changes there still seem to be things unfixed. I will take a closer look tomorrow and update this accordingly
